tpch/run.bash.ticat

```diff
@@ -0,0 +1,31 @@
+help = run TPCH benchmark
+abbr = r|R
```

Comment:
Too short, remove abbr

Reply:
refer to 
https://github.com/innerr/bench.tidb.ticat/blob/15d4457ffcd90b71eadbb546184a8d845ee67d93/tpcc/run.bash.ticat#L2

Reply:
refer to https://github.com/innerr/bench.tidb.ticat/blob/15d4457ffcd90b71eadbb546184a8d845ee67d93/tpcc/run.bash.ticat#L2

Reply:
refer to https://github.com/innerr/bench.tidb.ticat/blob/15d4457ffcd90b71eadbb546184a8d845ee67d93/tpcc/run.bash.ticat#L2

Reply:
refer to 
https://github.com/innerr/bench.tidb.ticat/blob/15d4457ffcd90b71eadbb546184a8d845ee67d93/tpcc/run.bash.ticat#L2

Reply:
Remove them both

Reply:
Second thought, might be OK, the path contains the workload name, small chance to conflict